frontends/main/validateEnv.js

```diff
@@ -21,6 +21,11 @@ const schema = yup.object().shape({
     .string()
     .oneOf(["true", "false"]),
   NEXT_PUBLIC_CSRF_COOKIE_NAME: yup.string().required(),
+  NEXT_PUBLIC_POSTHOG_PROJECT_ID: yup.string(),
+  NEXT_PUBLIC_POSTHOG_PERSONAL_API_KEY: yup.string(),
```

Comment:
Is `NEXT_PUBLIC_POSTHOG_PERSONAL_API_KEY` used for anything?

Reply:
No, and actually that shouldn't be in the build since it's the actually secret one. I pulled it (and the project ID; we don't use that either, PostHog figures it out just from the project key).